ext/drreg/drreg.h

```diff
@@ -434,6 +441,13 @@ drreg_status_t
 drreg_restore_app_values(void *drcontext, instrlist_t *ilist, instr_t *where, opnd_t opnd,
                          INOUT reg_id_t *swap);
 
+DR_EXPORT
+/**
+ * Restores all unreserved registers to their app values at \p where.
```

Comment:
Make it clear that this *includes* the flags.

Reply:
Done